cmd/list.go

```diff
@@ -32,10 +32,10 @@ func newListCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// listCmd represents the list command
 	var listCmd = &cobra.Command{
 		Use:   "list [repository]",
-		Short: "Lists the available Appsody stacks.",
-		Long: `Lists all the Appsody stacks available in your repositories. 
+		Short: "List the available Appsody stacks.",
+		Long: `List all the Appsody stacks available in your repositories. 
 
-An asterisk under the repository column denotes the default repository. An asterisk under the template column denotes the default stack template you will get when you initialise an Appsody project.`,
+An asterisk in the repository column denotes the default repository. An asterisk in the template column denotes the default template used, when you initialise an Appsody project.`,
```

Comment:
You don't need a comma after "used":
"... denotes the default template used when you initialise an Appsody project."
(Being picky, as a writer I'd say "... denotes the default template that is used when you initialise an Appsody project.", but since this needs to be short and won't be translated, it's not really necessary :) )

Reply:
I made the changes you requested as it's only the example explanations that we need to try keep short :)